<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stack_app_window</class>
 <widget class="QWidget" name="stack_app_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exemplul 1 cu PyQt5</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="stack_A_lbl">
       <property name="text">
        <string>Multimea A: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stack_A">
       <property name="text">
        <string>[1, 2, 3]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="regenerate_A_btn">
       <property name="text">
        <string>Generare multimea A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="stack_B_lbl">
       <property name="text">
        <string>Multimea B: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stack_B">
       <property name="text">
        <string>[4, 5, 6]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="regenerate_B_btn">
       <property name="text">
        <string>Generare multimea B</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="result">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="compute_btn">
       <property name="text">
        <string>Calculare expresie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
